samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/util/TextFormat.java

```diff
@@ -270,6 +270,7 @@ else if(upper.startsWith("WWW."))
    */
   public Integer indexOfIgnoreCase(String str, String searchString)
   {
+    Log log = LogFactory.getLog(TextFormat.class);
```

Comment:
I'm pretty sure this is all ok, my only comment would be for methods like this which may be used in tight loops and this is going to allocate a Log object for each call and create lots of GC (although it should never make it out of eden, so should be cheap to cleanup, but you still have the allocation).

For this class you could leave the logger as a class member (but not static) and just drop the main() method as that's the only reference to the logger that is static.

This isn't a -1, just a passing comment.